### PR TITLE
add validations

### PR DESCRIPTION
**Pull request by hidehiro98**
🙏 @db0sch 
Please check and merge my validations
I think these validation are minimum, but it is ok for at first

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Comment < ApplicationRecord
   belongs_to :request
   belongs_to :author, polymorphic: true
+
+  validates :content, presence: true
```
Comment (db0sch) [inline app/models/comment.rb]: you can add author and request, no? -> `validates :content, :author, :request, presence: true`

File: app/models/request.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class Request < ApplicationRecord
   belongs_to :user
   has_many :comments
+
+  validates :content, presence: true
```
Comment (db0sch) [inline app/models/request.rb]: you can add user -> `validates :content, :user, presence: true`

File: app/models/team.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
 class Team < ApplicationRecord
   has_many :users
+
+  validates :name, presence: true, uniqueness: true
```
Comment (db0sch) [inline app/models/team.rb:4]: :+1:

**Review by db0sch — APPROVED** [review]
Looks good to me.
I proposed you some modifications to have more complete validations. But still minimal.
As you want.

File: app/models/comment.rb (hunk 1 above)
Comment (hidehiro98) [reply app/models/comment.rb]: Exactly, but author and request are automatically set by our app not by user, so I don't think I need to validate them.
In this situation, how should I think about it? Is your validation good for double check?

File: app/models/request.rb (hunk 2 above)
Comment (hidehiro98) [reply app/models/request.rb]: It's same as above.

**Comment by db0sch** [comment]
I dont know about the fact that the user is not setting the author and request. 

I just know you don't want for some reason to save a comment without a author or request. So i think validating is better, as it will raise an error to the user, you'll be able to notice if there is something wrong in your code. 

What do you think?

**Comment by db0sch** [comment]
@hidehiro98
We can merge like this if you want. 
But first, you need to fix the conflicts.

**Comment by hidehiro98** [comment]
I see, yes, double check is needed.
I add validation as you suggested, thank you!

**Comment by hidehiro98** [comment]
@db0sch 
🙏

**Comment by db0sch** [comment]
looks good to me.

**Pull request merged**